src/event.js

```diff
@@ -582,16 +559,14 @@ function leverageNative( el, type, expectSync ) {
 			} else if ( saved.length ) {
 
 				// ...and capture the result
-				dataPriv.set( this, type, {
-					value: jQuery.event.trigger(
-
-						// Support: IE <=9 - 11+
-						// Extend with the prototype to reset the above stopImmediatePropagation()
-						jQuery.extend( saved[ 0 ], jQuery.Event.prototype ),
-						saved.slice( 1 ),
-						this
-					)
-				} );
+				dataPriv.set( this, type, jQuery.event.trigger(
```

Comment:
@gibson042 One thing to note - while #4350 which added this wrapping was mainly aimed at IE with its async `focus` handlers, there has been a report about this also affecting Chrome somehow:
https://github.com/jquery/jquery/issues/4350#issuecomment-482292263

I'm not sure if we can do anything about it due to the lack of data, though.